ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -1,33 +1,57 @@
-{% if vxlan_tunnel_interface is defined %}
+{% if vxlan_tunnel_interface is arista.avd.defined %}
 
 ## VXLAN Interface
 
 ### VXLAN Interface Summary
 
-#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.vxlan.source_interface }}
 
-#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port | arista.avd.default('4789') }}
 
-#### VLAN to VNI Mappings
+{% if vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address is arista.avd.defined %}
 
-| VLAN | VNI |
-| ---- | --- |
-{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans | arista.avd.natural_sort %}
-| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans[vlan].vni }} |
+#### EVPN MLAG Shared Router MAC : {{ vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address }}
+
+{% endif %}
+
+#### VLAN to VNI and Flood List Mappings
+
+| VLAN | VNI | Flood List |
+| ---- | --- | ---------- |
+{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan.vlans | arista.avd.natural_sort %}
+| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].vni }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | join('<br/>') if vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps is arista.avd.defined else 'N/A' }} |
 {%     else %}
-| N/A | N/A |
+| N/A | N/A | N/A |
 {%     endfor %}
-{% if vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs is defined and vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs is not none %}
+{% if vxlan_tunnel_interface.Vxlan1.vxlan.vrfs is arista.avd.defined and vxlan_tunnel_interface.Vxlan1.vxlan.vrfs is not none %}
```

Comment:
The test for `is not none` is included in `arista.avd.defined` and should be removed.

Reply:
addressed with commit dce6c86